.github/dependabot.yml

```diff
@@ -1,7 +1,24 @@
 version: 2
 
 updates:
+  - # Keep dev-v3 branch dependencies up to date, while Helm v3 is within support
+    package-ecosystem: "gomod"
+    target_branch: "dev-v3"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    groups:
+      k8s.io:
+        patterns:
+          - "k8s.io/api"
+          - "k8s.io/apiextensions-apiserver"
+          - "k8s.io/apimachinery"
+          - "k8s.io/apiserver"
+          - "k8s.io/cli-runtime"
+          - "k8s.io/client-go"
+          - "k8s.io/kubectl"
   - package-ecosystem: "gomod"
+    target_branch: "main"
```

Comment:
Added `target_branch` here to be explicit. I also hit issues with dependabot CI in https://github.com/helm/helm/pull/13508 when this wasn't set